app/cdap/components/ParsingDrawer/index.tsx

```diff
@@ -0,0 +1,91 @@
+import React, { useEffect, useState } from 'react';
+import Box from '@material-ui/core/Box';
+import ErrorOutlineIcon from '@material-ui/icons/ErrorOutline';
+import { Button } from '@material-ui/core';
+import { useStyles } from './styles';
+import { APPLY_BUTTON, IMPORT_SCHEMA, PARSING, PARSING_INFO_TEXT } from './constants';
+import ParsingPopupBody from './Components/ParsingPopupBody';
+import DrawerWidget from 'components/DrawerWidget';
+import ParsingHeaderActionTemplate from './Components/ParsingHeaderActionTemplate';
+
+const ParsingDrawer = (props) => {
```

Comment:
export as default function and remove function name

Reply:
Done